packages/contracts/development/contracts/RealityETH_ERC20-4.0.sol

```diff
@@ -734,16 +647,12 @@ contract RealityETH_ERC20_v3_0 is BalanceHolder_ERC20, IRealityETH_ERC20 {
             // We haven't yet got to the null hash (1st answer), ie the caller didn't supply the full answer chain.
             // Persist the details so we can pick up later where we left off later.
 
-            // If we know who to pay we can go ahead and pay them out, only keeping back last_bond
-            // (We always know who to pay unless all we saw were unrevealed commits)
-            if (payee != NULL_ADDRESS) {
-                _payPayee(question_id, payee, queued_funds);
-                queued_funds = 0;
-            }
+            // Pay out the latest payee, only keeping back last_bond which the next may have a claim on
+            _payPayee(question_id, payee, queued_funds);
+            queued_funds = 0;
```

Comment:
I think we don't need to reset the variable. It is no longer relevant, right?

Reply:
Nice catch, removed (with no bytecode changes!)